src/rules/selector-type-case/README.md

```diff
@@ -0,0 +1,58 @@
+# selector-type-case
+
+Specify lowercase or uppercase for html selector.
+
+```css
+    a {}
+/** ↑
+ * This type of selector */
+```
+
+## Options
+
+`string`: `"lower"|"upper"`
+
+### `"lower"`
+
+The following patterns are considered warnings:
+
+```css
+A { color: #fff; }
```

Comment:
Can you use empty blocks for all the examples to bring it inline with how we [normally demo](https://github.com/stylelint/stylelint/blob/master/src/rules/selector-no-type/README.md) selector rules. We like to keep each rule's docs focused on the patterns being tested.